R/tm_t_crosstable.R

```diff
@@ -167,6 +174,7 @@ tm_t_crosstable <- function(label = "Cross Table",
   )
 }
 
+# UI function for the cross-table module
```

Comment:
@averissimo in other PRs @kartikeyakirar put `@noRd` and `@keywords internal` for those `ui_` and `server_` functions. Can we have them here as well, so it's unified... sorry?